llvm/test/Transforms/LoopVectorize/RISCV/strided-accesses.ll

```diff
@@ -357,6 +674,22 @@ define void @single_stride_ptr_iv(ptr %p, i64 %stride) {
 ; CHECK:       exit:
 ; CHECK-NEXT:    ret void
 ;
+; CHECK-UF2-LABEL: @single_stride_ptr_iv(
+; CHECK-UF2-NEXT:  entry:
+; CHECK-UF2-NEXT:    br label [[LOOP:%.*]]
+; CHECK-UF2:       loop:
+; CHECK-UF2-NEXT:    [[I:%.*]] = phi i64 [ 0, [[ENTRY:%.*]] ], [ [[NEXTI:%.*]], [[LOOP]] ]
+; CHECK-UF2-NEXT:    [[PTR:%.*]] = phi ptr [ [[P:%.*]], [[ENTRY]] ], [ [[PTR_NEXT:%.*]], [[LOOP]] ]
+; CHECK-UF2-NEXT:    [[X0:%.*]] = load i32, ptr [[PTR]], align 4
+; CHECK-UF2-NEXT:    [[Y0:%.*]] = add i32 [[X0]], 1
+; CHECK-UF2-NEXT:    store i32 [[Y0]], ptr [[PTR]], align 4
+; CHECK-UF2-NEXT:    [[PTR_NEXT]] = getelementptr inbounds i8, ptr [[PTR]], i64 [[STRIDE:%.*]]
+; CHECK-UF2-NEXT:    [[NEXTI]] = add i64 [[I]], 1
+; CHECK-UF2-NEXT:    [[DONE:%.*]] = icmp eq i64 [[NEXTI]], 1024
+; CHECK-UF2-NEXT:    br i1 [[DONE]], label [[EXIT:%.*]], label [[LOOP]]
+; CHECK-UF2:       exit:
+; CHECK-UF2-NEXT:    ret void
```

Comment:
Could you add a COMMON prefix so we have a single set of check lines when the IR is the same for both?